pwndbg/aglib/disasm/instruction.py

```diff
@@ -119,18 +123,13 @@ class SplitType(Enum):
 # in the disasm view output (see 'pwndbg.color.disasm.instruction()'), as well as for
 # Pwndbg commands like "nextcall" that need to know the instructions target to set breakpoints
 class PwndbgInstruction:
-    def __init__(self, cs_insn: CsInsn | None) -> None:
+    def __init__(self, cs_insn: CsInsn) -> None:
         self.cs_insn: CsInsn = cs_insn
         """
         The underlying Capstone instruction, if present.
```

Comment:
Please update the docstring. Iiuc previously we could have a `None` instruction here and now we drop this, right?